mlbench/refimpls/pytorch/utils/helper.py

```diff
@@ -1,22 +1,52 @@
 import time
+import itertools
+from utils import log
 
 
 class Timeit(object):
-    def __enter__(self):
+    def __init__(self):
         self.t = time.time()
         self._cumu = 0
-        return self
-
-    def __exit__(self, type, value, traceback):
-        pass
+        self._paused = False
 
     def pause(self):
         self._cumu += time.time() - self.t
```

Comment:
`pause()` should do nothing if `self._paused`  is already `True`, to prevent bugs.